test/integration/load/load_test.go

```diff
@@ -139,39 +131,27 @@ func TestValidateState(t *testing.T) {
 	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Minute)
 	defer cancel()
 
-	var validator stateValidator
-
-	t.Log("Validating the state file")
 	switch *osType {
 	case "linux":
-		validator, err = validate.CreateLinuxValidator(ctx, clientset, config, namespace, *cniType, *restartCase)
+		linuxValidator, err := validate.CreateLinuxValidator(ctx, clientset, config, namespace, *cniType, *restartCase)
 		if err != nil {
 			t.Fatal(err)
 		}
+		if err := linuxValidator.Validate(ctx); err != nil {
+			t.Fatal(err)
+		}
+
 	case "windows":
-		validator, err = validate.CreateWindowsValidator(ctx, clientset, config, namespace, *cniType, *restartCase)
+		windowsValidator, err := validate.CreateWindowsValidator(ctx, clientset, config, namespace, *cniType, *restartCase)
 		if err != nil {
 			t.Fatal(err)
 		}
+		if err := windowsValidator.Validate(ctx); err != nil {
+			t.Fatal(err)
+		}
 	default:
 		t.Fatalf("unknown os type %s", *osType)
 	}
-
-	err = validator.ValidateStateFile(ctx)
```

Comment:
looks like this got removed, maybe on accident?

Reply:
Since this is linux specific, I moved it linux validate file instead of load test.